### PR TITLE
fix: 🐛 Correct validation for removing asset requirements

### PR DESCRIPTION
**Pull request by prashantasdeveloper**
### Description

* Correct the condition to check if given requirement id exists in existing requirements for an Asset 

### JIRA Link

[DA-384](https://polymath.atlassian.net/browse/DA-384)

### Checklist

- [ ] Updated the Readme.md (if required) ?

### CODE REVIEW

File: src/api/procedures/removeAssetRequirement.ts [hunk 1]
```diff
@@ -37,7 +37,7 @@ export async function prepareRemoveAssetRequirement(
 
   const { requirements } = await query.complianceManager.assetCompliances(rawTicker);
 
-  if (requirements.filter(({ id: rawId }) => u32ToBigNumber(rawId).eq(reqId)).length) {
+  if (!requirements.filter(({ id: rawId }) => u32ToBigNumber(rawId).eq(reqId)).length) {
```
Comment (monitz87) [inline src/api/procedures/removeAssetRequirement.ts:40]: `find` makes more sense here

**Review by monitz87 — CHANGES_REQUESTED** [review]

File: src/api/procedures/removeAssetRequirement.ts (hunk 1 above)
Comment (prashantasdeveloper) [reply src/api/procedures/removeAssetRequirement.ts]: Since we don't actually need the value, I guess `some` fits better here

**Review by monitz87 — APPROVED** [review]

**Review by VictorVicente — APPROVED** [review]

**Pull request merged**